config/routes.rb

```diff
@@ -7,6 +7,9 @@
   get '/breeds/:name', to: "breeds#show", as: "breed"
   get '/cats', to: "cats#index"
   get '/cats/:id', to: "cats#show", as: "cat"
+  post '/cart_items', to: "cart_items#create"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.